boost.lwr

```diff
@@ -18,13 +18,13 @@
 #
 
 category: baseline
-configure: ./bootstrap.sh --prefix=$prefix --libdir=$prefix/lib/
+configure: cd .. && ./bootstrap.sh --prefix=$prefix --libdir=$prefix/lib/
 depends:
 - python
 - libbzip
-install: ./bjam --threading=multi --layout=tagged link=shared -j$makewidth install
-install_static: './bjam cxxflags=''-fPIC'' --threading=multi --layout=tagged link=static
-  -j$makewidth install
+install: cd .. && ./bjam --threading=multi --layout=tagged link=shared -j$makewidth install --build-dir=build
+install_static: 'cd && ./bjam cxxflags=''-fPIC'' --threading=multi --layout=tagged link=static
+  -j$makewidth install --build-dir=build
   '
```

Comment:
it's right here. I agree it looks weird/wrong, but I didn't touch the formatting.